src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/DynamicMethod.CoreCLR.cs

```diff
@@ -0,0 +1,305 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Runtime.Loader;
+using System.Text;
+using System.Threading;
+using static System.Runtime.CompilerServices.RuntimeHelpers;
+
+namespace System.Reflection.Emit
+{
+    public sealed partial class DynamicMethod : MethodInfo
+    {
+        private RuntimeType[] parameterTypes;
```

Comment:
The naming convention is to prefix field names with underscore: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

(I should have said to strip the `m` prefix, not `m_` prefix.)